_pages/en/esdc-it-strategy.md

```diff
@@ -52,43 +47,125 @@ But a complex and uncertain world rewards an empirical cycle of trying, observin
+This involves:
+
+- Increased focus on outcomes over advance planning activities
+- Increasing the frequency of deployment in production to receive feedback from end-users
+- Allowing IT teams to engage directly with end-users
+- Increased awareness of policy makers on the impacts of policies to end-users and IT systems
+
+<!--markdownlint-enable MD029 MD036-->
+
+### Long Term Outcomes (Impact)
+
+The long term impacts of the IT Strategy is to create the following two conditions:
+
+<!--markdownlint-disable MD029 MD036-->
+**1) Technology is a shared accountability between stakeholders**
+
+Lines of businesses _share the costs and risks_ of using technology with the IT branch.
+Before an IT-enabled initiative is launched, the business problem to solve with technology _is clear_, business experts _fully participate_ in the execution of the initiative in partnership with their technology colleagues, and _rapid feedback_ from end-users is not only promoted, it is _enabled_.
+
+The business vs IT paradigm is replaced with a team partnership paradigm.
+
+**2) The four (4) key metrics as identified in the [DevOps Research and Assessment (DORA) institute research](https://www.devops-research.com/research.html) for software delivery & operational performance are continuously improved**
+
+- _Lead time:_ the time code changes take to go from check-in to release in production
+- _Deployment frequency:_ the rate at which software is deployed to production or released to end users
+- _Change fail:_ the change failure rate measured by how often deployment failures occur in production that require immediate remedy
+- _Time to restore:_ the time it takes from detecting a user impacting incident to remediating it
+<!--markdownlint-enable MD029 MD036-->
+
+### Strategy Map
+
+The above outcomes will make use of this [Strategy Map diagram](strategy-summary.html) to guide technology teams towards the short, medium, and long term outcomes.
+The strategy map accepts that individual teams may be at different maturity levels and so accepts that teams may have different starting points.
+The content in this Strategy map is guided by the [DevOps Research and Assessment (DORA)](https://cloud.google.com/devops) 21 capabilities.
+
+Each block corresponding in the diagram represents a distinct strategy to move ESDC towards its ultimate Digital Transformation goal: become a key stakeholder in the Canadian Government as a Platform.
 
-## Immediate Actions
+### IITB Teams
 
-To get started, the following [Strategies and Actions](strategies-actions.html) need to be applied for all projects and teams.
+Efforts and outcomes produced by [activities of all IT teams](http://dialogue/grp/IITB-DGIIT-Gov-New-Nouveau/Documents/Departmental%20IMIT%20Plan/19-20%20Plans/IITB%20Moving%20Forward%20v2.docx) continue and all contribute towards the same long term impact.
```

Comment:
Not sure it's necessary to include this. IITB Way Forward is already mentioned at the beginning. If we keep, maybe not in a distinct (rather short) section.

Reply:
The intent of this section is to expose the work of other IITB teams within the logic model. So that IT Strategy team is not seen as the sole agent of change.

Reply:
Will keep current wording